doc/building.md

```diff
@@ -1092,16 +1092,33 @@ apt install g++-aarch64-linux-gnu gcc-aarch64-linux-gnu
 
   * Create chroot on the *build* system, configuring it for *target* system:
 ```
-sudo qemu-debootstrap --arch=arm64 --verbose \
-       --include=fakeroot,build-essential,libx11-dev,libxext-dev,libxrender-dev,libxrandr-dev,libxtst-dev,libxt-dev,libcups2-dev,libfontconfig1-dev,libasound2-dev,libfreetype6-dev,libpng12-dev \
-       --resolve-deps jessie /chroots/arm64 http://httpredir.debian.org/debian/
+sudo qemu-debootstrap \
+  --arch=arm64 \
+  --verbose \
+  --include=fakeroot,symlinks,build-essential,libx11-dev,libxext-dev,libxrender-dev,libxrandr-dev,libxtst-dev,libxt-dev,libcups2-dev,libfontconfig1-dev,libasound2-dev,libfreetype6-dev,libpng-dev \
+  --resolve-deps \
+  buster \
```

Comment:
I'm guessing `--resolve-deps buster` is the full option; if so, it should be on a single line, otherwise it reads very strange.

Reply:
Not really: `buster` is the `SUITE`, it is the argument on its own.

Reply:
Correct, ```buster``` is the suite (= Debian release).

You might want to use ```stable``` instead of ```buster```, then you don't need to adjust that part all the time as ```buster``` is a symlink to ```buster``` on the Debian FTP servers.

Reply:
Thanks. I'd rather leave `buster`, because we know that would work with the package list the line above requires. I believe I had some problems with `libpng-dev` before. So changing the suite later would also test the include package list.